packages/@storefront/flux-capacitor/src/core/actions/creators.ts

```diff
@@ -715,7 +715,7 @@ namespace ActionCreators {
    * @return {Actions.ReceiveMoreProducts}          - Action with products.
    */
   export function receiveMoreProducts(res: Results) {
-    return (state: Store.State): Actions.ReceiveMoreProducts => {
+    return (_: Store.State): Actions.ReceiveMoreProducts => {
```

Comment:
We can, but I like still having the type there so that if a new developer comes on and needs to start using the parameter again, they know how it works already.